CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.0
+
+- Migrating to null-safety
+
```

Comment:
Do not change version in PR since it will be consolidated during release. Because the version number at the time of merging this PR can't be predicted.

Please remove changes in CHANGELOG